rule/templates/_rule.js

```diff
@@ -8,28 +8,37 @@
 // Rule Definition
 //------------------------------------------------------------------------------
 
-module.exports = function(context) {
+module.exports = {
+    meta: {
+        docs: {
+            description: "<%= desc %>",
+            category: "Fill me in",
+            recommended: false
+        },
+        fixable: null,  // or "code" or "whitespace"
+        schema: [
+            // fill in your schema
+        ]
+    },
 
-    // variables should be defined here
+    create: function(context) {
 
-    //--------------------------------------------------------------------------
-    // Helpers
-    //--------------------------------------------------------------------------
+        // variables should be defined here
 
-    // any helper functions should go here or else delete this section
+        //----------------------------------------------------------------------
+        // Helpers
+        //----------------------------------------------------------------------
```

Comment:
I wonder if the helpers should go here or all the way above, outside the exports expressions, and before the "Rule Definition" delimiter?

Just wondering because it seems like we have some inconsistency with regards to that in our codebase at the moment.
For example, the rules `no-empty-character-class` and `accessor-pairs` put the helpers outside the exports assignment, but the rules `array-bracket-spacing` and `camelcase` put it inside.